src/CovidAppointmentTable.js

```diff
@@ -193,7 +193,7 @@ function RestrictionNotifier({ entry }) {
                 }
             >
                 <Typography className={classes.restrictionWarning}>
-                    Important Eligibility Notice
+                    Information about eligibility
```

Comment:
Was this change from "Important Eligibility Notice" to "Information about eligibility" intended?
I didn't see it explained.

Reply:
It was recommend in the [#ux channel by Daniel](https://macovidvaccines.slack.com/archives/C01N7C2G2KF/p1614795429083700)

Reply:
'k. It should appear in the commit message when this is merged to `master`, whether by way of being in the summary of this PR or by being in an additional individual commit you further push to the PR now.

Reply:
I edited the PR comment at the top.